SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UNR-4281/SpatialTestPlayerControllerHandover.cpp

```diff
@@ -0,0 +1,274 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
We're trying to move away from having bug numbers in file paths, could you remove it or replace it with something more descriptive?